packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -0,0 +1,227 @@
+"""Support functions for the Measurement Plug-In Client generator."""
+
+import keyword
+import os
+from enum import Enum
+from typing import Dict, Tuple
+
+import grpc
+from google.protobuf import descriptor_pool
+from google.protobuf.type_pb2 import Field
+from ni_measurement_plugin_sdk_service._internal.grpc_servicer import frame_metadata_dict
+from ni_measurement_plugin_sdk_service._internal.stubs.ni.measurementlink.measurement.v2 import (
```

Comment:
Should these internal imports be replaced by the client_support wrapper module?

Reply:
Only, `stubs` in `_internal` is exposed to user. The usage of `frame_metadata_dict` in internal is restricted to generator code and it is not exposed in the generated code, which we give to the user.